tests/functional/workflows/python_pip/test_packager.py

```diff
@@ -519,7 +519,7 @@ def test_does_fail_on_invalid_local_package(self, tmpdir, osutils, pip_runner):
         pip, runner = pip_runner
         appdir, builder = self._make_appdir_and_dependency_builder(reqs, tmpdir, runner)
         requirements_file = os.path.join(appdir, "requirements.txt")
-        pip.set_return_tuple(0, (b"Processing ../foo\n" b"  Link is a directory," b" ignoring download_dir"), b"")
```

Comment:
Wonder if we should keep thins so we know we don't break back compat? Or make this output from pip parameterized to test both cases?

Reply:
Reverted those tests to keep using the old output string. Updated the unit tests to test a mixture of reading both new and old.